pkg/kf/commands/install/kf/kf.go

```diff
@@ -85,6 +95,51 @@ func Install(ctx context.Context, containerRegistry string) error {
 	return nil
 }
 
+func waitForKfDeployments(ctx context.Context) error {
+	ctx, cancel := context.WithTimeout(ctx, time.Minute)
+	defer cancel()
+
+	for _, deploymentName := range []string{"controller", "webhook"} {
+		Logf(ctx, "waiting for %s deployment to be available...", deploymentName)
+		err := wait.ExponentialBackoff(
+			wait.Backoff{
+				Duration: time.Second,
+				Steps:    10,
+				Factor:   1,
+			}, func() (bool, error) {
+				output, err := Kubectl(
+					ctx,
+					"get",
+					"deployments",
+					deploymentName,
+					"--namespace", v1alpha1.KfNamespace,
+					"--output=json",
+				)
+				if err != nil {
+					return false, err
+				}
+				deployment := appsv1.Deployment{}
```

Comment:
I'd rather use the real rest client instead because it's fewer moving parts. Alternatively this could use a different output type in the request to just get back the correct value.

Reply:
So I went w/ kubectl so I didn't add a config dependency to the installer AND to keep w/ the theme of printing out the kubectl commands used.

I wanted to use the --field-selector, but the conditions are a slice so I wasn't sure how to do so.